cmd/ipfs/init.go

```diff
@@ -9,6 +9,7 @@ import (
 	"path"
 
 	context "context"
+
```

Comment:
while we're modifying this, lets go ahead and drop the named import (holdover from `"golang.org/x/net/context"`) and move it up with the rest of the stdlib